src/main/java/com/meilisearch/sdk/HttpClient.java

```diff
@@ -13,7 +13,7 @@
 import com.meilisearch.sdk.json.JsonHandler;
 import java.util.Collections;
 
-/** The HTTP requests for the different functions to be done through Meilisearch */
+/** HTTP client to be used for the different functions to be performed through Meilisearch */
```

Comment:
```suggestion
/** HTTP client used for API calls to Meilisearch */
```

I don't think it ads value to add more information. On the contrary, it confuses more.